Makefile

```diff
@@ -1,8 +1,31 @@
 all: build
 
-include csi.mk
+CSI_SPEC := spec.md
+CSI_PROTO := csi.proto
 
-build: $(CSI_PROTO) $(CSI_GOSRC)
+# the target for building the temporary csi protobuf file.
```

Comment:
Hi @jieyu,

As I was the first to create comments in the Makefiles for this project, is this your personal preference or part of the project's style guidelines? What you see is my preference, but I will change it in accordance with a posted style manual. Thanks!